docs/HowTo/Configure/Consensus-Protocols/QBFT.md

```diff
@@ -0,0 +1,354 @@
+
+To start a node on a QBFT private network, use the
+[`--genesis-file`](../../../Reference/CLI/CLI-Syntax.md#genesis-file) option to specify the custom
+genesis file.
+
+### Extra data
+
+The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization
+scheme used in Ethereum. To generate the `extraData` RLP string for inclusion in the genesis file,
+use the [`rlp encode`](../../../Reference/CLI/CLI-Subcommands.md#rlp) Besu subcommand.
+
+!!! example
+
+    ```bash
+    besu rlp encode --from=toEncode.json --type=QBFT_EXTRA_DATA
+    ```
+
+Where the `toEncode.json` file contains a list of the initial validators, in ascending order. To
+write the validator address and copy it to the `toEncode.json` file, use the
+[`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) Besu
+subcommand. For example:
+
+!!! example "One Initial Validator in `toEncode.json` file"
+
+    ```json
+    [
+     "9811ebc35d7b06b3fa8dc5809a1f9c52751e1deb"
+    ]
+    ```
+
+Copy the RLP encoded data to the `extraData` property in the genesis file.
+
+### Block time
+
+When the protocol receives a new chain head, the block time (`blockperiodseconds`) timer starts.
+When `blockperiodseconds` expires, the round timeout (`requesttimeoutseconds`) timer starts and
+the protocol proposes a new block.
+
+If `requesttimeoutseconds` expires before adding the proposed block, a round change occurs, with
+the block time and timeout timers reset. The timeout period for the new round is two times
+`requesttimeoutseconds`. The timeout period continues to double each time a round fails to add a
+block.
+
+Usually, the protocol adds the proposed block before reaching `requesttimeoutseconds`. A new round
+then starts, resetting the block time and round timeout timers. When `blockperiodseconds`
+expires, the protocol proposes the next new block.
+
+!!! warning
+
+    If more than 1/3 of validators stop participating, new blocks can no longer be created and 
+    `requesttimeoutseconds` doubles with each round change. The quickest method
+    to resume block production is to restart all validators, which resets `requesttimeoutseconds` to
+    its genesis value.
+
+Once `blockperiodseconds` is over, the time from proposing a block to adding the block is
+small (usually around one second) even in networks with geographically dispersed validators.
+
+#### Tuning block timeout
+
+To tune the block timeout for your network deployment:
+
+1. Set `blockperiodseconds` to your desired block time and `requesttimeoutseconds` to two times
+   `blockperiodseconds`.
+1. Reduce `requesttimeoutseconds` until you start to see round changes occurring.
+1. Increase `requesttimeoutseconds` to the value where round changes are no longer occurring.
+
+!!! tip
+
+    View [`TRACE` logs](../../../Reference/API-Methods.md#admin_changeloglevel) to see round change
+    log messages.
+{!global/Config-Options.md!}
+
+## Adding and removing validators
```

Comment:
I moved this section for Clique and IBFT 2.0 to the separate "Adding Validators" page in a recent PR (now merged). Should probably do the same for QBFT for consistency.

Reply:
Done